debian/copyright

```diff
@@ -0,0 +1,52 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Sirupsen/logrus
+Source: https://github.com/Sirupsen/logrus
+
+Files: *
+Copyright: 2014 Simon Eskildsen
+License: MIT
+
+Files: debian/*
+Copyright: 2014 Tianon Gravi <admwiggin@gmail.com>
+License: BSD-2-clause
```

Comment:
This should probably be `MIT` as well, to match upstream, and you need to add yourself to the line above. :+1:

```
Files: debian/*
Copyright: 2014 Tianon Gravi <admwiggin@gmail.com>,
           2015 Jessie The Bomb
License: MIT
```